PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -597,6 +609,19 @@ class Benchmark {
             Realm.dispose(magicFrame);
     }
 
+    warmUpNetworkCache() {
+        if (!isInBrowser)
+            return;
+        let filePromises = this.plan.files.map((file) => fileLoader.load(file));
+        let preloads = [];
+        if (this.plan.preload) {
+            for (let prop of Object.getOwnPropertyNames(this.plan.preload))
+                preloads.push(this.plan.preload[prop]);
+        }
+        preloads = preloads.map((file) => fileLoader.load(file));
+        return Promise.all([filePromises, preloads]);
```

Comment:
This is passing array of array of promises, so this function will not wait for all promises' resolution.

Reply:
Fixed.